src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.SymbolReferenceFinder.cs

```diff
@@ -167,7 +170,84 @@ private List<SymbolReference> DeDupeAndSortReferences(List<SymbolReference> allR
                 return GetMatchingTypes(searchScope, name, arity, inAttributeContext, symbols, hasIncompleteParentMember);
             }
 
-            private async Task<IEnumerable<SearchResult<ITypeSymbol>>> GetTypeSymbols(
+            internal async Task FindNugetReferencesAsync(List<Reference> allReferences)
+            {
+                if (allReferences.Count > 0)
+                {
+                    // Only do this if none of the project searches produced any results. We always
```

Comment:
// Only do this if none of the project searches produced any *exact* results.

Otherwise you have to go digging around to confirm that.